cmd/loop/loopout.go

```diff
@@ -25,10 +27,10 @@ var loopOutCommand = cli.Command{
 	Optionally a BASE58/bech32 encoded bitcoin destination address may be
 	specified. If not specified, a new wallet address will be generated.`,
 	Flags: []cli.Flag{
-		cli.Uint64Flag{
+		cli.StringFlag{
```

Comment:
`StringSliceFlag` so we don't need to parse ourselves?

Reply:
Never heard of it. Added! To bad there seems to be no Uint64SliceFlag

Reply:
Added, but doesn't work as expected. For every channel, the complete flag needs to be added. Left as is with more compact csv

Reply:
Feature request https://github.com/urfave/cli/issues/1134

Reply:
You can specify `channel={chan1,chan2}` in braces then it'll parse correctly, but a single comma separated flag may be more user-friendly, happy with either.

Reply:
Ok didn't know that. But the braces pattern isn't very common indeed, so leaving as is